src/content/developers/docs/nodes-and-clients/nodes-as-a-service

```diff
@@ -0,0 +1,95 @@
+lang: en
+sidebar: true
+sidebarDepth: 2
+---
+
+## Introduction {#Introduction}
+
+Running your own [Ethereum node](/developers/docs/nodes-and-clients/#what-are-nodes-and-clients) can be challenging, especially when getting started or while scaling fast. There are a [number of services](/developers/docs/nodes-and-clients-nodes-as-a-service/#popular-node-services) that run optimized node infrastructures for you, so you can focus on developing your application or product instead. We'll explain how node services work, the pros and cons for using them and list providers if you are interested in getting started. 
+
+## Prerequisites {#prerequisites}
+If you don't already have an understanding of what nodes and clients are check out the [Nodes and Clients page](/developers/docs/nodes-and-clients/).
+
+## How do Node Services Work? {#how-do-node-services-work}
+
+Node services are providers run distributed node clients behind the scenes for you.
+
+These services typically provide an API key that you can use to write to and read from the blockchain, and often including access to [Ethereum testnets](/developers/docs/networks) in addition to the mainnet.
+
+Some services offer you your own dedicated node that they manage for you, while others use load balancers to distribute activity across nodes.
+
+Almost all node services are extremely easy to integrate with, involving one line changes in your code to swap out your self hosted node, or even switch between the services themselves.
+
+Often times node services will run a variety of [node clients](/developers/docs/nodes-and-clients/#clients) and [types](/developers/docs/nodes-and-clients/#node-types), allowing you to access full and archive nodes in addition to client specific methods in one API.
+
+It's important to note that node services do not and should not store your private keys or information.
+
+## What are the benefits of using a Node Service? {#benefits-of-using-a-node-service}
+
+The main benefit for using a node service is not having to spend engineering time maintaining and managing nodes yourself. This allows you to focus on building your product rather than having to worry about infrastructure maintenance.
+
+Running your own nodes can be very expensive from storage to bandwidth to valuable engineering time. Things like spinning up more nodes when scaling, upgrading nodes to the latest versions, and ensuring state consistency, can detract from building and spending resources on your desired web3 product.
+
+## What are the cons of using a Node Service? {#cons-of-using-a-node-service}
+
+By using a node service you are centralizing the infrastructure aspect of your product. For this reason, projects that hold decentralization to the upmost importance might prefer self-hosting nodes rather than outsourcing to a 3rd party.
+
+You can read more about the benefits of running your own node [here](/developers/docs/nodes-and-clients/#benefits-to-you).
```

Comment:
We try to avoid links like "here" or "find out more". Where possible make the link copy descriptive like:

`Read more about the [benefits of running your own node](/developers/docs/nodes-and-clients/#benefits-to-you)`